binderhub/builder.py

```diff
@@ -290,8 +290,14 @@ def escape(s):
         ).replace('_', '-').lower()
 
         if self.settings['use_registry']:
-            image_manifest = await self.registry.get_image_manifest(*'/'.join(image_name.split('/')[-2:]).split(':', 1))
-            image_found = bool(image_manifest)
+            for _ in range(3):
+                try:
+                    image_manifest = await self.registry.get_image_manifest(*'/'.join(image_name.split('/')[-2:]).split(':', 1))
+                    image_found = bool(image_manifest)
+                    break
+                except tornado.curl_httpclient.CurlError:
+                    app_log.warning("Tornado HTTP Timeout error: Failed to get image manifest for %s", image_name)
```

Comment:
```suggestion
                    app_log.exception("Tornado HTTP Timeout error: Failed to get image manifest for %s", image_name)
```

Reply:
The self-suggest :D

Reply:
Sometimes it's just easier than pulling my changes and faffing with vscode :joy: